index.html

```diff
@@ -439,6 +447,69 @@ <h2>Legend</h2>
        }
      }
 
+     // Sorting
+     function setupSorting() {
+       const table = document.querySelector("table");
+
+       const headers = table.querySelectorAll("th");
+
+       headers.forEach((header, columnIndex) => {
+         if (!header.classList.contains('sortable')) {
+           return;
+         }
+         const button = document.createElement("button");
+         const isInitDescending = header.classList.contains("more-info");
+         const sortNumerically = isInitDescending;
+         button.setAttribute("aria-label", `Sort by ${header.textContent}`);
+         button.textContent = header.textContent;
+         header.setAttribute("aria-label", header.textContent);
+         header.textContent = '';
+         header.append(button);
+         if (isInitDescending) {
+           header.setAttribute("aria-sort", "descending");
+         }
+
+         button.onclick = () => {
+           const isAscending = header.getAttribute("aria-sort") !== "ascending";
```

Comment:
This is a pretty confusing assignment.  I think that your intent is to say that the sort should be ascending when the current state is not ascending, but the negation and naming here makes it look backwards.  Maybe:

```suggestion
           const sortAscending = header.getAttribute("aria-sort") !== "ascending";
```